SimGeneral/MixingModule/python/mixObjects_cfi.py

```diff
@@ -221,6 +221,14 @@
 
 from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import hgceeDigitizer, hgchefrontDigitizer
 
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+run2_GEM_2017.toModify( theMixObjects,
+    mixSH = dict(
+        input = theMixObjects.mixSH.input + [ cms.InputTag("g4SimHits","MuonGEMHits") ],
+        subdets = theMixObjects.mixSH.subdets + [ 'MuonGEMHits' ],
+        crossingFrames = theMixObjects.mixSH.crossingFrames + [ 'MuonGEMHits' ]
```

Comment:
@slava77 I dumped the step3 cfg and I see input and subdects modified (not sure if this is the correct way to check):
process.theMixObjects = cms.PSet(
mixCH = cms.PSet(
crossingFrames = cms.untracked.vstring(),
input = cms.VInputTag(cms.InputTag("g4SimHits","CastorFI"), cms.InputTag("g4SimHits","EcalHitsEB"), cms.InputTag("g4SimHits","EcalHitsEE"), cms.InputTag("g4SimHits","EcalHitsES"), cms.InputTag("g4SimHits","HcalHits"),
cms.InputTag("g4SimHits","ZDCHITS")),
subdets = cms.vstring('CastorFI',
'EcalHitsEB',
'EcalHitsEE',
'EcalHitsES',
'HcalHits',
'ZDCHITS'),
type = cms.string('PCaloHit')
),
mixHepMC = cms.PSet(
input = cms.VInputTag(cms.InputTag("generatorSmeared"), cms.InputTag("generator")),
makeCrossingFrame = cms.untracked.bool(False),
type = cms.string('HepMCProduct')
),
mixSH = cms.PSet(
crossingFrames = cms.untracked.vstring('MuonCSCHits',
'MuonDTHits',
'MuonRPCHits',
'MuonGEMHits'),
input = cms.VInputTag(cms.InputTag("g4SimHits","MuonCSCHits"), cms.InputTag("g4SimHits","MuonDTHits"), cms.InputTag("g4SimHits","MuonRPCHits"), cms.InputTag("g4SimHits","TrackerHitsPixelBarrelHighTof"), cms.InputTag("g4SimHits","TrackerHitsPixelBarrelLowTof"),
cms.InputTag("g4SimHits","TrackerHitsPixelEndcapHighTof"), cms.InputTag("g4SimHits","TrackerHitsPixelEndcapLowTof"), cms.InputTag("g4SimHits","TrackerHitsTECHighTof"), cms.InputTag("g4SimHits","TrackerHitsTECLowTof"), cms.InputTag("g4SimHits","TrackerHitsTIBHighTof"),
cms.InputTag("g4SimHits","TrackerHitsTIBLowTof"), cms.InputTag("g4SimHits","TrackerHitsTIDHighTof"), cms.InputTag("g4SimHits","TrackerHitsTIDLowTof"), cms.InputTag("g4SimHits","TrackerHitsTOBHighTof"), cms.InputTag("g4SimHits","TrackerHitsTOBLowTof"),
cms.InputTag("g4SimHits","MuonGEMHits")),
subdets = cms.vstring('MuonCSCHits',
'MuonDTHits',
'MuonRPCHits',
'TrackerHitsPixelBarrelHighTof',
'TrackerHitsPixelBarrelLowTof',
'TrackerHitsPixelEndcapHighTof',
'TrackerHitsPixelEndcapLowTof',
'TrackerHitsTECHighTof',
'TrackerHitsTECLowTof',
'TrackerHitsTIBHighTof',
'TrackerHitsTIBLowTof',
'TrackerHitsTIDHighTof',
'TrackerHitsTIDLowTof',
'TrackerHitsTOBHighTof',
'TrackerHitsTOBLowTof',
'MuonGEMHits'),
type = cms.string('PSimHit')
),
mixTracks = cms.PSet(
input = cms.VInputTag(cms.InputTag("g4SimHits")),
makeCrossingFrame = cms.untracked.bool(False),
type = cms.string('SimTrack')
),
mixVertices = cms.PSet(
input = cms.VInputTag(cms.InputTag("g4SimHits")),
makeCrossingFrame = cms.untracked.bool(False),
type = cms.string('SimVertex')
)
)